src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -1191,5 +1289,41 @@ MakeSolutionGroupConvImplicitGemmXdlops(const miopen::conv::ProblemDescription&
 #endif
 }
 
+template <typename InvokerFactoryMakerNHWC>
+ConvSolution
+MakeSolutionGroupConvImplicitGemmNCHWXdlops(const miopen::conv::ProblemDescription& problem,
+                                            InvokerFactoryMakerNHWC&& invoker_factory_maker_ndhwc)
```

Comment:
Maybe `solution_maker_ndhwc`?
```suggestion
                                            InvokerFactoryMakerNHWC&& solution_maker_ndhwc)
```